cmd/list/oidcprovider/cmd.go

```diff
@@ -41,11 +41,23 @@ var Cmd = &cobra.Command{
 	Run: run,
 }
 
+var args struct {
+	oidcConfigId string
+}
+
 func init() {
 	flags := Cmd.Flags()
 	flags.SortFlags = false
 	output.AddFlag(Cmd)
 	ocm.AddOptionalClusterFlag(Cmd)
+
+	flags.StringVarP(
+		&args.oidcConfigId,
+		"oidc-config-id",
+		"i",
+		"",
```

Comment:
@hunterkepley Do we want to include a `usage` string here to help the users?

Reply:
Oh, there should be one, on line 59

Reply:
My mistake. You're absolutely right...not sure how I missed that!